src/apps/loyalty/containers/login/index.tsx

```diff
@@ -53,7 +53,7 @@ class Login extends React.Component<LoginProps, LoginState> {
 
           {form.csrfToken && <input type="hidden" name="_csrf" value={form.csrfToken} />}
 
-          <Button block>Log In</Button>
+          <Button block style={{flexDirection: "column"}}>Log In</Button>
```

Comment:
The facebook and twitter buttons need to be set to the default which is why I added it here but I can still move it to the base button component and set the `flex-direction` to row for the twitter and facebook button components